RocketChat/Rocket.Chat.ReactNative
# fix: remove blocks when updating an old message

## Proposed changes


With the implementation of the alert when a user is mentioned and is not in the channel, a bug appeared where, after pressing the action button, these buttons are not hidden.

## Issue(s)	


## How to test or reproduce


- Go to a channel where you are an owner, leader or moderator
- Mention someone outside the channel
- It will appear some buttons as "add them", "do nothing" or "let them know"
- After you press the buttons, this message will be updated and the buttons will hide.

## Screenshots

#### Before

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/84040a3a-8c5b-4e2b-8b97-932f513b0733

#### After

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/16e24fb8-1948-4dbf-beac-187a79d7ba95

## Types of changes




## Checklist



## Further comments

[TC-1092]
